{{ cookiecutter.package_name }}/.github/workflows/tox-tests.yml

```diff
@@ -0,0 +1,104 @@
+# GitHub Actions workflow for testing and continuous integration.
+#
+# This file performs testing using tox and tox.ini to define and configure the test environments.
+
+name: Python Tests
+
+on: [push, pull_request]
```

Comment:
Maybe also a good idea to include cron job (say, weekly)?

Reply:
agreed. if nothing else, then as an example how to do it.